server/graphql/types/src/generic_errors.rs

```diff
@@ -25,7 +25,7 @@ impl StockLineReducedBelowZero {
 #[Object]
 impl StockLineReducedBelowZero {
     pub async fn description(&self) -> &'static str {
-        "Stock line exist in new outbound shipments. "
+        "Stock line reduced below zero."
```

Comment:
Reusing so made the description more generic

Reply:
I've hit this error a few times now when trying to repack. Is there some way we can prevent users trying to repack into numbers that cause this?

Reply:
Will see what I can do in the other PR for error handling 🤔